content/en/docs/concepts/architecture/controller.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Controllers
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+In robotics and automation, a _control loop_ is
+a non-terminating loop that regulates the state of a system.
+
+Here is one example of a control loop: a thermostat in a room.
+
+When you set the temperature, that's telling the thermostat
+about your *desired state*. The actual room temperature is the
+*current state*. The thermostat acts to bring the current state
+closer to the desired state, by turning equipment on or off.
+
+In Kubernetes, a _controller_ is a control loop that watches the
+state of your
```

Comment:
hello @sftim. Great work! A few thoughts 😃,
- Did you use the controller glossary term/tooltip on this page?
- I think it is important to connect this page to the other Controller pages under Workloads (for example, https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/). Another page that seems related is, https://kubernetes.io/docs/concepts/overview/components/

Reply:
Good idea. That was one of my planned tasks for the epic-shaped [set of changes](https://github.com/kubernetes/website/pull/15373) I had in mind.

Reply:
In the reviewed revision https://kubernetes.io/docs/concepts/overview/components/#kube-controller-manager/ already hyperlinks to this new page, because the glossary tooltip for “controller” servers as a hyperlink.
I added a _what's next_ link as well.

For now, I put some glossary tooltips for “controller” into the workload resource pages (Deployment, DaemonSet etc). I expect to put in a related PR that adds reference documentation for the associated controllers (Deployment controller, DaemonSet controller etc).
In that related PR I plan to make text like “Deployment controller” hyperlink to the relevant reference documentation, so the tooltips will go again. I'm happy to have those tooltips in place in the meantime.